app/assets/javascripts/require_recaptcha.js

```diff
@@ -0,0 +1,7 @@
+function recaptcha_verified () {
+  document.querySelector('.recaptcha_submit').removeAttribute('disabled')
+}
+
+function recaptcha_expired () {
+  document.querySelector('.recaptcha_submit').setAttribute('disabled', 'disabled')
+}
```

Comment:
I wasn't quite sure where to add this javascript for enabling/disabling the submit button based on the status of the recaptcha.

Reply:
I think this is the appropriate place to put this since `recaptcha_tags` is expecting a `callback` and `expired_ callback` function. Though it would be ideal to have this handled on the React side in [startup scripts](https://github.com/ifmeorg/ifme/tree/main/client/app/startup) since we want to use the Rails asset pipeline for JS as little as possible. We could explore doing this in another PR :)